jobs/scripts/fs-integration/fs-integration.sh

```diff
@@ -98,6 +95,16 @@ virsh capabilities
 # run the tests
 #
 
+set +x
+
 EXTRA_VARS="${TEST_EXTRA_VARS}" make "${TEST_TARGET}"
+ret=$?
+
+EXTRA_VARS="${TEST_EXTRA_VARS}" make statedump
+
+pushd "/tmp/sit.${BACKEND}_statedump"
+tar -zcvf /tmp/statedump.tar.gz .
```

Comment:
I'm not sure if it wouldn't be better to include the directory in the tar, so that when you untar you don't get all the files created at the current directory.

Reply:
Good point, updated in the next version.